backend/src/packages/orders/order.controller.ts

```diff
@@ -410,11 +415,19 @@ class OrderController extends Controller {
       user: UserEntityObjectWithGroupT | null;
     }>,
   ): Promise<ApiHandlerResponse<OrderResponseDto>> {
+    const userId = options.user?.id ?? null;
+    const customerPhone =
+      options.body.customerPhone ?? options.user?.phone ?? null;
+    const customerName =
+      options.body.customerName ?? options.user?.firstName ?? null;
```

Comment:
contollers should be clean. pass user to service and then decide what to save into db

rewrite to:
const { firstName = null, phone = null } = user ?? {};
const nameInOrder = customerName ?? firstName;
const phoneInOrder = customerPhone ?? phone;